doc/specs/#1043 - Set the initial position of the Terminal/spec.md

```diff
@@ -0,0 +1,98 @@
+---
+author: Kaiyu Wang KaiyuWang16/kawa@microsoft.com
+created on: 2019-09-03
+last updated: 2019-09-03
+issue id: #1043
+---
+
+# Set the initial position for terminal
+
+## Abstract
+
+This spec is for task #1043 �Be able to set an initial position for the terminal�. It goes over the details of a new feature that allows users to set the initial position and size of the terminal. Expected behavior and design of this feature is included. Besides, future possible follow-up works are also addressed.
+
+## Inspiration
+
+The idea is to allow users to set the initial position of the Terminal when they launch it, prevent the Terminal from appearing on unexpected position (e.g. outside of the screen bounds). We are also going to let users choose to maximize the window when they launch it.
+
+## Solution Design
+
+For now, the Terminal window is put on a default initial position. The program uses CW_USEDEFAULT in the screen coordinates for top-left corner. We have two different types of window � client window and non-client window. However, code path for window creation (WM_CREATE message is shared by the two types of windows) are almost the same for the two types of windows, except that there are some differences in calculation of the width and height of the window.
+
+Two new properties should be added in the json settings file:
+
+**initialPosition**: string. This sets the initial horizontal and vertical position of the top-left corner of the window. This property follows a structure: "X value, Y value" and has following rules:
+
+1. All spaces will be ignored.
+
+2. Both X value and Y values are optional. If anyone of them is missing, or the value is invalid, system default value will be used. Examples:
+
+       ", 1000" equals to (default, 1000)
+       "1000, " equals to (1000, default)
+       "," equals to (default, default)
+       "abc, 1000" equals to (default, 1000)
+
+**launchMode**: string. Determine the launch mode. There are two modes for now
+                         1. maximize: the window will be maximized when launch.

```

Comment:
I'd probably add a newline above these two lines, and de-indent them. Right now, in the rendered markdown, they just flow into the above line.

Reply:
changed